<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contract_delete</class>
 <widget class="QDialog" name="contract_delete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>75</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete contract</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter id:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_contract_delete">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contract">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
